src/errors.js

```diff
@@ -1,12 +1,22 @@
-class EncryptionError extends Error {}
-class EncryptionJsonError extends EncryptionError {}
-class DecryptionError extends Error {}
-class DecryptionIntegrityError extends DecryptionError {}
-class DecryptionSignatureError extends DecryptionError {}
-class DecryptionTimeTravelError extends DecryptionError {}
-class DecryptionExpiredError extends DecryptionError {}
-class DecryptionGunzipError extends DecryptionError {}
-class DecryptionJsonError extends DecryptionError {}
+class NamedError extends Error {
+  constructor(message, fileName, lineNumber) {
+    super(message, fileName, lineNumber);
```

Comment:
Just use and pass `...args`. No need to be explicit.

Reply:
ok